docs/mlir_interoperation.ipynb

```diff
@@ -0,0 +1,310 @@
+{
```

Comment:
<p>Maybe mention that this prints MLIR in generic form. Clearly, beginners won't understand, but for people familiar with MLIR, this is crucial info.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xdslproject/xdsl/pull/394/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>